ide/slf4j.jdk14/external/slf4j-jdk14-1.7.36-license.txt

```diff
@@ -1,10 +1,10 @@
 Name: SLF4J
 Version: 1.7.36
-License: MIT-slf4j
+License: MIT-slf4j-22
```

Comment:
The source license for 1.7.36 is here:
https://raw.githubusercontent.com/qos-ch/slf4j/v_1.7.36/LICENSE.txt